### PR TITLE
PWX-31962: Op-integration test for dmthin basic install and uninstall

### PR DESCRIPTION
**Pull request by nrevanna**
**What this PR does / why we need it**:

**Which issue(s) this PR fixes** (optional)
Closes #

**Special notes for your reviewer**:

```
--- PASS: TestStorageClusterBasicDMthin (1472.68s)
    --- PASS: TestStorageClusterBasicDMthin/BasicInstallDmthinWithMetadataDevice (768.89s)
    --- PASS: TestStorageClusterBasicDMthin/BasicInstallDmthinWithMetadataAndKVDBDevice (703.78s)
PASS
==================================================
--- PASS:
    BasicInstallDmthinWithMetadataDevice []
    BasicInstallDmthinWithMetadataAndKVDBDevice []
--- SKIP:
--- FAIL:
==================================================

DONE 3 tests in 1472.953s
Tests passed
```

### CODE REVIEW

File: test/integration_test/basic_dmthin_test.go [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+		// Construct StorageCluster
+		err := ci_utils.ConstructStorageCluster(cluster, ci_utils.PxSpecGenURL, ci_utils.PxSpecImages)
+		require.NoError(t, err)
+
+		// Deploy PX and validate
+		cluster = ci_utils.DeployAndValidateStorageCluster(cluster, ci_utils.PxSpecImages, t)
+
+		// Verify we see PXStoreV2 in pxctl status output
+		out, _, err := ci_utils.RunPxCmd("/usr/local/bin/pxctl status| grep 'PX-StoreV2'|wc -l")
```
Comment (zoxpx) [inline test/integration_test/basic_dmthin_test.go:115]: note, you'll get the same "count effect" w/ `pxctl status | grep -c PX-StoreV2`

File: pkg/util/test/util.go [hunk 2]
```diff
@@ -1093,9 +1093,10 @@ func validateStorageClusterPods(
 // and makes sure its there, if dmthin misc-args annotation is found
 func validateDmthinOnPxNodes(cluster *corev1.StorageCluster) error {
 	listOptions := map[string]string{"name": "portworx"}
-	cmd := "cat /etc/pwx/config.json | grep px-storev2"
+	cmd := "cat /etc/pwx/config.json | grep -i px-storev2"
```
Comment (zoxpx) [inline pkg/util/test/util.go:1096]: nit: same effect w/ `grep -i px-storev2 /etc/pwx/config.json`

File: pkg/util/test/util.go [hunk 3]
```diff
@@ -1093,9 +1093,10 @@ func validateStorageClusterPods(
...
+	miscArgAnnotation := cluster.Annotations["portworx.io/misc-args"]
 
-	if !strings.Contains(cluster.Annotations["portworx.io/misc-args"], "-T px-storev2") {
+	if !strings.Contains(strings.ToLower(miscArgAnnotation), strings.ToLower("-T px-storev2")) {
```
Comment (zoxpx) [inline pkg/util/test/util.go:1099]: nit: same effect w/ `...Contains(strings.ToLower(miscArgAnnotation), "-t px-storev2")`

**Review by zoxpx — APPROVED** [review]
I think this looks OK, thanks Naveen -- some minor remarks below

**Comment by nrevanna** [comment]
Thanks Zoran. Addressed your comments.

**Pull request merged**